docs/docs/writing-plugins/the-rules-api/testing-plugins.mdx

```diff
@@ -82,6 +82,27 @@ For Approach #4, you should use `setup_tmpdir()` to set up BUILD files.
 
 To use `run_rule_with_mocks`, pass the `@rule` as its first arg, then `rule_args=[arg1, arg2, ...]` in the same order as the arguments to the `@rule`.
 
+For example:
+
+```python
+from pants.engine.rules import collect_rules, rule
+from pants.testutil.rule_runner import run_rule_with_mocks
+
+
+@rule
+async def int_to_str(i: int) -> str:
+    return str(i)
+
+
+def rules():
+    return collect_rules()
```

Comment:
what does this add to the example here? (is it required to work?)

Reply:
I thought that since you are required to collect the rules in the module, it would make sense to include it here (as if it's stored in another module). But I have now removed it to keep the test case concise.